plugin/pkg/admission/broker/authsarcheck/admission.go

```diff
@@ -33,12 +33,13 @@ import (
 )
 
 const (
-	pluginName = "BrokerAuthSarCheck"
+	// PluginName is name of admission plug-in
+	PluginName = "BrokerAuthSarCheck"
```

Comment:
why make this public?

Reply:
To be consistent with every other admission plugin in this project and in kubernetes, plus I'm working on some stuff to embed service catalog in another API server and means I don't need to copy the string name :)

Reply:
What does such a use look like?

Reply:
Trying to put together a stripped down extensible API server that can be used for only CRDs, API extensions, and service catalog - IMO almost a holy grail of extensible architecture for APIs :)